utils/config/daos_server.yml

```diff
@@ -170,11 +170,16 @@
 ## Number of hugepages to allocate for use by NVMe SSDs
 #
 ## Specifies the number (not size) of hugepages to allocate for use by NVMe
-## through SPDK. This indicates the number to be used for each spawned
-## I/O Engines, so the total will be this number * number of I/O Engines.
-## Default system hugepage size will be used.
+## through SPDK. Note that each target requires 1 GiB of hugepage space, so
+## this value needs to represent the total amount of hugespace required for
+## all targets across all engines on host, divided by the system hugepage size.
+#
+## Example: (2 engines * (16 targets/engine * 1GiB)) / 2MiB hugepage size = 16834
```

Comment:
I don't think this is correct for the 2.0 branch. I just checked the code: https://github.com/daos-stack/daos/blob/release/2.0/src/control/server/server_utils.go#L181

On release/2.0, this is still a per-engine value, so I think this example just needs to remove the engine multiplier.

Reply:
Good catch, I missed that. thanks @mjmac